test/quantization/fx/test_quantize_fx.py

```diff
@@ -4461,6 +4495,21 @@ def forward(self, x):
                 x = self.bn(x)
                 return x
 
+        class LinearBnLeakyReLUModel(torch.nn.Module):
+            def __init__(self, with_bn=True):
+                super(LinearBnLeakyReLUModel, self).__init__()
```

Comment:
nit: `super().__init__()`

Reply:
is this test module the same as the test module in test_fuse_linear_bn_leaky_relu_eval

Reply:
> nit: `super().__init__()`

This is to align with other sub-tests in the same test case: https://github.com/pytorch/pytorch/blob/299fbbccec68f9e8e0fa19a1066a8e179e9ab633/test/quantization/fx/test_quantize_fx.py#L4430

Reply:
> is this test module the same as the test module in test_fuse_linear_bn_leaky_relu_eval? should we use a single definition and add that to common_quantization.py? https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantization.py

Looks like they are doing similar things. Maybe I can remove `test_fuse_linear_bn_leaky_relu_eval`. Here I align with the sub-test of `LinearReLU`.
What do you mean by single definition in common_quantizaiton.py? Like this? https://github.com/pytorch/pytorch/blob/299fbbccec68f9e8e0fa19a1066a8e179e9ab633/torch/testing/_internal/common_quantization.py#L1323